CMakeLists.txt

```diff
@@ -130,45 +211,50 @@ aux_source_directory(${CMAKE_CURRENT_SOURCE_DIR}/src SRCS)
 list(REMOVE_ITEM SRCS ${MAIN_FILE})
 list(APPEND SRCS ${SHINONOME_SRCS})
 if(CMAKE_SYSTEM_NAME MATCHES "Windows")
-  list(APPEND SRCS "${CMAKE_CURRENT_SOURCE_DIR}/resources/player.rc")
+	list(APPEND SRCS "${CMAKE_CURRENT_SOURCE_DIR}/resources/player.rc")
 endif()
 add_library(${PROJECT_NAME}_Static STATIC ${SRCS})
 
 # executable
 add_executable(${PROJECT_NAME} MACOSX_BUNDLE ${MAIN_FILE})
 add_executable(${PROJECT_NAME}_com ${MAIN_FILE})
 foreach(i ${PROJECT_NAME} ${PROJECT_NAME}_com)
-  target_link_libraries(${i} ${EASYRPG_PLAYER_LIBRARIES_ALL})
-  add_dependencies(${i} liblcf ${PROJECT_NAME}_Static)
+	target_link_libraries(${i} ${EASYRPG_PLAYER_LIBRARIES_ALL})
+	add_dependencies(${i} ${PROJECT_NAME}_Static)
+	if(PLAYER_BUILD_LIBLCF)
+		add_dependencies(${i} liblcf)
+	endif()
 endforeach()
 install(TARGETS ${PROJECT_NAME} EXPORT ${PROJECT_NAME} DESTINATION bin)
 
 # manpage
+# Handled as extra target "man"
+
 set(MAN_NAME easyrpg-player.6)
 find_program(A2X_EXECUTABLE NAMES a2x a2x.py)
 if(NOT A2X_EXECUTABLE STREQUAL "A2X_EXECUTABLE-NOTFOUND")
-  add_custom_command(OUTPUT resources/${MAN_NAME}
-    COMMAND ${A2X_EXECUTABLE} -a player_version="0.4.1" -f manpage -D ${CMAKE_CURRENT_BINARY_DIR}/resources ${CMAKE_CURRENT_SOURCE_DIR}/resources/${MAN_NAME}.adoc
-    DEPENDS resources/${MAN_NAME}.adoc
-    COMMENT "(Re-)building manpage ${MAN_NAME}"
-    VERBATIM)
-  if(${CMAKE_SYSTEM_NAME} MATCHES "Linux")
-    add_custom_target(man ALL DEPENDS resources/${MAN_NAME})
-    install(FILES resources/${MAN_NAME} DESTINATION share/man/man6)
-  else()
-    add_custom_target(man DEPENDS resources/${MAN_NAME})
-  endif()
+	add_custom_command(OUTPUT resources/${MAN_NAME}
+		COMMAND ${A2X_EXECUTABLE} -a player_version="0.4.1" -f manpage -D ${CMAKE_CURRENT_BINARY_DIR}/resources ${CMAKE_CURRENT_SOURCE_DIR}/resources/${MAN_NAME}.adoc
+		DEPENDS resources/${MAN_NAME}.adoc
+		COMMENT "(Re-)building manpage ${MAN_NAME}"
+		VERBATIM)
+	if(${CMAKE_SYSTEM_NAME} MATCHES "Linux")
+		add_custom_target(man ALL DEPENDS resources/${MAN_NAME})
+		install(FILES resources/${MAN_NAME} DESTINATION share/man/man6)
+	else()
+		add_custom_target(man DEPENDS resources/${MAN_NAME})
+	endif()
 else()
-  # no a2x, distribution archive?
-  if(EXISTS ${CMAKE_CURRENT_SOURCE_DIR}/resources/${MAN_NAME})
-    install(FILES resources/${MAN_NAME} DESTINATION share/man/man6)
-  endif()
+	# no a2x, distribution archive?
+	if(EXISTS ${CMAKE_CURRENT_SOURCE_DIR}/resources/${MAN_NAME})
+		install(FILES resources/${MAN_NAME} DESTINATION share/man/man6)
+	endif()
 endif()
 
 # CPack
 set(CPACK_GENERATOR "ZIP" "TGZ")
 if(${CMAKE_SYSTEM_NAME} MATCHES "Windows")
-  list(APPEND CPACK_GENERATOR "NSIS")
+	list(APPEND CPACK_GENERATOR "NSIS")
```

Comment:
generating a windows installer for the portable player was always a bit off IMO, so maybe we should remove that option now